### ISSUE TITLE
Blocks

### ISSUE THREAD

**Issue by xSuperr** [Refaltor77/CustomItem#2]
A way to add Unmapped blocks.

**Comment by morcheysha77** [comment]
> A way to add Unmapped blocks.

Yes, it exists but this is not the purpose of the plugin and it is much more complicated but a register plugin already has blocks, I will redirect you to poggit.

**Comment by Refaltor77** [comment]
block by Refaltor and Natof soon

**Comment by Refaltor77** [comment]
https://github.com/Big-Potatoes/CustomItem 🥔
